### PR TITLE
Updates copyright to 2015

### PR DESCRIPTION
**Pull request by Uladox**
Uses the grep -rl '2014' ./ | xargs sed -i 's/2012/2015/g' command in the src directory

### CODE REVIEW

File: src/generated/rpg_eventcommand.h [hunk 1]
```diff
@@ -126,8 +126,8 @@ namespace RPG {
 				ConditionalBranch_B = 13310,
 				TerminateBattle = 13410,
 				ShowMessage_2 = 20110,
-				ShowChoiceOption = 20140,
-				ShowChoiceEnd = 20141,
+				ShowChoiceOption = 20150,
+				ShowChoiceEnd = 20151,
```
Comment (Ghabry) [inline src/generated/rpg_eventcommand.h]: .

Comment (Uladox) [inline src/generated/rpg_eventcommand.h]: Oh, I better fix that.

**Comment by Ghabry** [comment]
I don't really see a reason in updating copyrights. We just putted the first copyright year there (when the lib switched to MIT license). And copyright lasts for at least 70 years.
And you forgot updating the templates in the template directory. And then just run the generate.py script.

**Comment by Uladox** [comment]
Go point, got caught up in seeing something easy to update and reacted prematurely.

**Pull request closed without merging**